zombienets/turing/moonbase.toml

```diff
@@ -0,0 +1,35 @@
+[settings]
+provider = "native"
+timeout = 1000
+
+[relaychain]
+default_command = "../polkadot/target/release/polkadot"
+chain = "rococo-local"
+
+[[relaychain.nodes]]
+name = "alice"
+
+[[relaychain.nodes]]
+name = "bob"
+
+[[parachains]]
+id = 2114
+cumulus_based = true
+chain = "turing-dev"
+
+[parachains.collator]
+name = "turing-col-1"
+command = "./target/release/oak-collator"
+ws_port = 9946
+rpc_port = 8855
+
+[[parachains]]
+id = 1000
+cumulus_based = true
+chain = "moonbase-local"
```

Comment:
Their name is "moonbeam-dev"

Reply:
moonbeam-dev is a single chain for development.
It's not a parachain.

Reply:
oh I see. If that’s the case, let’s revert it to moonbase-col-1.